docs/Translation.md

```diff
@@ -62,6 +62,7 @@ You can find translation packages for the following languages:
 - Turkish (`tr`): [ismailbaskin/aor-language-turkish](https://github.com/ismailbaskin/aor-language-turkish)
 - Ukrainian (`uk`): [vitivs/aor-language-ukrainian](https://github.com/vitivs/aor-language-ukrainian)
 - Vietnamese (`vi`): [kimkha/aor-language-vietnamese](https://github.com/kimkha/aor-language-vietnamese)
+- Arabic ( `ع` ): [aymendhaya/aor-language-arabic ](https://github.com/aymendhaya/aor-language-arabic)
```

Comment:
please add in the list in alphabetical order, and use arabic in the animated GIF of your repository